site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -189,6 +193,112 @@ To prepare the manifest file, do the following:
+
+1. Prepare data.
+   For more information,
+   refer on how to [prepare the dataset](#prepare-the-dataset).
+1. Open the bucket and select **Upload/Download**.
+1. Drag and drop files or folders, or select **Browse files** to upload your data.
+
+Alternatively, you can use the [Backblaze CLI](https://www.backblaze.com/docs/cloud-storage-command-line-tools)
+or any S3-compatible tool like the AWS CLI with B2 endpoints.
+
+### <a name="access-permissions-b2">Access permissions</a>
+
+To access your B2 bucket from CVAT, you need to create Application Keys:
+
+1. In the Backblaze console, go to **App Keys**.
+1. Select **Add a New Application Key**.
+1. Configure the key:
+   - **Name of Key**: Enter a descriptive name (e.g., "CVAT Access").
+   - **Allow access to Bucket(s)**: Select the specific bucket you created, or choose **All** for access to all buckets.
+   - **Type of Access**: Select **Read and Write** for full access, or **Read Only** if you only need to import data.
+   - **Allow List All Bucket Names**: Enable if you want to list all buckets.
+   - **File name prefix**: (Optional) Restrict access to specific file prefixes.
+   - **Duration**: (Optional) Set an expiration time for the key.
+1. Select **Create New Key**.
+1. **Important**: Save the **keyID** and **applicationKey** immediately.
+   The **applicationKey** is only shown once and cannot be retrieved later.
+
+{{% alert title="Warning" color="warning" %}}
+Store your **applicationKey** securely. It will only be displayed once
+during creation and cannot be recovered later.
+{{% /alert %}}
+
+For more information, consult
+[B2 Application Keys](https://www.backblaze.com/docs/cloud-storage-application-keys).
+
+### Attach Backblaze B2 storage
+
+To attach B2 storage to CVAT, do the following:
+
+1. Log into CVAT.
+1. In CVAT, on the top menu select **Cloud storages** > on the opened page select **+**.
+
+Fill in the following fields:
+
+<!--lint disable maximum-line-length-->
+
+| CVAT                    | Backblaze B2 |
+| ----------------------- | ------------ |
+| **Display name**        | Preferred display name for your storage. |
+| **Description**         | (Optional) Add description of storage. |
+| **Provider**            | From drop-down list select **AWS S3** (B2 is S3-compatible). |
+| **Bucket name**         | Name of your B2 bucket. |
+| **Authentication type** | Select **Key id and secret access key pair**. |
+| **Access key ID**       | Enter the **keyID** from your [B2 Application Key](#access-permissions-b2). |
+| **Secret access key**   | Enter the **applicationKey** from your [B2 Application Key](#access-permissions-b2). |
+| **Region**              | Enter your B2 bucket region (e.g., `us-west-004`). You can find this in your bucket details. To get the region: go to **B2 Cloud Storage** > **Buckets** > select your bucket > look for **Endpoint** (e.g., `s3.us-west-004.backblazeb2.com`) - the region is the part between `s3.` and `.backblazeb2.com`. |
```

Comment:
I don't think this is needed; with S3 this field is used to construct the endpoint URL, but since we're overriding the URL anyway, I don't think setting the region will have any effect.